scripts/performance/tracer.py

```diff
@@ -59,17 +59,37 @@ def is_console_exporter_enabled() -> bool:
     '''Return whether the console exporter has been enabled.'''
     return tracing_state_manager.get_console_exporter_enabled()
 
+# ParamSpec was added in Python 3.10, so we need to use typing_extensions for older versions.
```

Comment:
How many machines do we have running python <3.10? Maybe we can use this for now but update all machines to latest python and remove it after?

Reply:
The only one I found was the alpine queues

Reply:
I'm in favour of moving this to a separate PR down the line, just in case I've missed anywhere else that needs python 3.9